src/pages/learn.tsx

```diff
@@ -104,7 +113,12 @@ export default ({ data, location }: Props) => {
           {pages.slice(5)}
         </ul>
       </nav>
-      <article className="article-reader" dangerouslySetInnerHTML={{ __html: activePage.html }} />
+      <div>
+        <article className="article-reader" dangerouslySetInnerHTML={{ __html: activePage.html }} />
+        <a href={`https://github.com/nodejs/website-prototype/tree/master/src/documentation/${activePage.relativePath}`}>
```

Comment:
Do we maybe want to drop them directly into the editor?

```suggestion
        <a href={`https://github.com/nodejs/website-prototype/edit/master/src/documentation/${activePage.relativePath}`}>
```

Reply:
That's a quite good idea.

About link style: definitely we should change it. I just used the naked a tag because couldn't see similar link with style. For now we can use the style of links from TOC list
<img width="260" alt="image" src="https://user-images.githubusercontent.com/2481234/52780276-1b956700-304a-11e9-9cc2-af4cbd66e9c1.png">